### DISCUSSION TITLE
Auto completion doesn't work m1 mac

### DISCUSSION THREAD

**Discussion by miraculusik** [nvim-neorg/neorg#730] [Q&A] [answered]
```lua
require("nvim-treesitter.install").compilers = { "gcc-12" }
require("neorg").setup({
	load = {
		["core.defaults"] = {},
		["core.norg.completion"] = { config = { engine = "nvim-cmp" } },
		["core.norg.concealer"] = {},
		["core.norg.dirman"] = {
			config = {
				workspaces = {
					notes = "~/notes",
				},
			},
		},
	},
})

```
I am not getting any error message but auto completion is not working, note: other functions working nicely.
Note: I added cmp `source={name={"neorg"}}`

**Comment by max397574** [comment]
what exactly are you trying to do?

**Reply by miraculusik** [reply]
I want the autocomplete window to open in any typing operation. 
In treesitter showcase he has a lot of auto completion options. such as creating list, table, codeblock etc. I don't have any of these features. I'm using nvim-cmp.

**Reply by max397574** [reply]
are you sure you did add the neorg source?

**Reply by miraculusik** [reply]
> are you sure you did add the neorg source?

Yes, still doesn't work.I updated the question, thanks

**Comment by miraculusik** [comment, marked as answer]
finally i solved this problem.
this problem is caused by auto-pairs plugin. Disable auto-pairs for norg files
